libethcore/SealEngine.h

```diff
@@ -154,5 +154,18 @@ u256 calculateEthashDifficulty(
 
 u256 calculateGasLimit(ChainOperationParams const& _chainParams, BlockHeader const& _bi,
     u256 const& _gasFloorTarget = Invalid256);
+
+
+class NoReward : public NoProof
+{
+public:
+    static std::string name() { return "NoReward"; }
+    static void init();
+    u256 blockReward(u256 const& _blockNumber) const override
+    {
+        (void)_blockNumber;
```

Comment:
(Nit) Can you remove this if you don't include the arg name in the arg list?

Reply:
Then it wont override

Reply:
Of course this shouldn't affect overriding.
What Nils meant is
```
 u256 blockReward(u256 const&) const override
 {
    return 0;
 }
```